pkg/alertmanager/multitenant.go

```diff
@@ -479,10 +483,16 @@ func (am *MultitenantAlertmanager) ServeHTTP(w http.ResponseWriter, req *http.Re
 	userAM, ok := am.alertmanagers[userID]
 	am.alertmanagersMtx.Unlock()
 
-	if !ok || !userAM.IsActive() {
-		http.Error(w, "no Alertmanager for this user ID", http.StatusNotFound)
+	if !ok {
+		http.Error(w, "no Alertmanager for this user ID, Alertmanager not configured", http.StatusNotFound)
+		return
+	}
+
+	if !userAM.IsActive() {
+		http.Error(w, "Alertmanager is not active, revise its configuration", http.StatusNotAcceptable)
```

Comment:
I'm not sure about the message here. Internally we activate/deactivate the alertmanager because we can't completely remove it, but from the final user perspective it's the same exact thing as "Alertmanager has not been configured".

Reply:
You're completely right - paused is only used when the Alertmanager is gone. I'll have to rethink this one.

Reply:
Ok, let's address this and then we can go.

Reply:
What do you think to rollback the logic, so we can unblock this PR and get it merged?